Makefile

```diff
@@ -36,12 +36,15 @@ test-container:
 lint: test-container
 	# E111 indentation is not a multiple of four
 	# E121 continuation line under-indented for hanging indent
+	# E127 continuation line over-indented for visual indent
 	# E401 multiple imports on one line
 	# E402 module level import not at top of file
 	# E501 line too long (N > 79 characters)
 	# E722 do not use bare 'except'
 	# W293 blank line contains whitespace
-	$(CONTAINER_ENGINE) run --rm -v `pwd -P`:`pwd -P` $(REPO_NAME):test /bin/sh -c "cd `pwd`; flake8 --ignore E111,E121,E114,E401,E402,E501,E722,W293 src/"
+	# W503 line break before binary operator
+	# W504 line break after binary operator
+	$(CONTAINER_ENGINE) run --rm -v `pwd -P`:`pwd -P` $(REPO_NAME):test /bin/sh -c "cd `pwd`; flake8 --ignore E111,E121,E127,E114,E401,E402,E501,E722,W293,W503,W504 src/"
```

Comment:
We're going to start going with pep8 with 120 chars max (just discussed in leads call).  I won't block this PR on not adding these but just know it will have to be taken care of eventually.

Reply:
sure. this line is originally yours https://github.com/openshift/managed-cluster-validating-webhooks/commit/72ef3198e55392c85702bc0bfc35cdf19a0ea607#diff-b67911656ef5d18c4ae36cb6741b7965R43